### ISSUE TITLE
Easier way to go to Home page from the docs

### ISSUE THREAD

**Issue by theoludwig** [adonisjs/v6-docs#138]
Hey! :wave:

Most often to go to the documentation, I go to https://adonisjs.com/, then click on "Docs" or "Get started".
Sometimes I need to go back to home page, for example to go to the blog, or to go to https://packages.adonisjs.com/, but from the docs, it is not easily possible.

### Proposal

The logo Adonis on the header on the left, instead to go the home page of docs, it should go to https://adonisjs.com/.
The "home page of the docs", is already easily accessible with the Preface accordion and "Introduction".

What do you think? :smile:
It's a small thing, but can improve a bit the User Experience.

**Comment by thetutlage** [comment]
Yeah it will be nice to have a menu with dropdown that has links to everything directly from the docs. Adding it to my todo and will do it.
